### PR TITLE
Add integrations test with ActionDispatch::ShowExceptions

### PR DESCRIPTION
**Pull request by robertomiranda**

### CODE REVIEW

**Comment by Bodacious** [comment]
Hi @robertomiranda 

This looks great. I've updated the spec setup/teardown to reset the Rails app config back to its previous state. 
https://github.com/errm/promenade/pull/24/commits/17778f857e444a898f543f29515f96fd79f9357f

WDYT?

File: spec/integration/show_exceptions_middleware_spec.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    expect(response.status).to eq(500)
+    expect(response.body).to eq("500 Internal Server Error Page\n")
+  end
+
+  private
+    attr_accessor :initial_env_config
+
+    def enable_show_exceptions
+      self.initial_env_config = Rails.application.env_config
```
Comment (Bodacious) [inline spec/integration/show_exceptions_middleware_spec.rb:53]:
```suggestion
      self.initial_env_config = Rails.application.env_config.dup
```

**Comment by robertomiranda** [comment]
> This looks great. I've updated the spec setup/teardown to reset the Rails app config back to its previous state.
https://github.com/errm/promenade/commit/17778f857e444a898f543f29515f96fd79f9357f

perfect, good catch 👌

**Comment by Bodacious** [comment]
Sorry @robertomiranda, I didn't realise merging my PR would automatically close yours. Are you able to re-open on `master`?

**Comment by robertomiranda** [comment]
@Bodacious re-opened as new PR https://github.com/errm/promenade/pull/25

**Pull request closed without merging**